master/internal/api_experiment.go

```diff
@@ -1469,6 +1469,27 @@ func (a *apiServer) parseAndMergeContinueConfig(expID int, overrideConfig string
 			fmt.Sprintf("override config must have single searcher type got '%s' instead", overrideName))
 	}
 
+	// Determine which workspace the experiment is in.
+	wkspName := activeConfig.Workspace()
+	if wkspName == "" {
+		wkspName = "Uncategorized"
+	}
+	ctx := context.TODO()
+	w, err := workspace.WorkspaceByName(ctx, wkspName)
+	if err != nil {
+		return nil, false,
+			fmt.Errorf("error getting workspace %s: %w", activeConfig.Workspace(), err)
+	}
+	// Merge the config with the optionally specified invariant config specified by task config
+	// policies.
+	configWithInvariantDefaults, err := configpolicy.MergeWithInvariantExperimentConfigs(ctx,
+		w.ID, mergedConfig)
+	if err != nil {
+		return nil, false,
+			fmt.Errorf("error merging invariant experiment configs: %w", err)
```

Comment:
should either of these errors be a `status` return with error?

Reply:
Hmm, great observation! I think this error is fine, but I changed the workspace retrieval error message to `status` with `codes.Internal`!